client/e2e/workflows/bceidbusiness/industry_user.spec.ts

```diff
@@ -153,9 +153,6 @@ test.describe("Test Workflow industry_user", () => {
       selectOperatorPage.buttonSubmit,
     );
 
-    // Add short timeout to mitigate the Firefox text rendering issue causing spurious screenshot failures
```

Comment:
Did you already adding an assertion? Could just check the url or something to slow it down

Reply:
Yeah I tried a few things and did a timeout as a last resort. Shon suggested adding it to ignored diffs though I was trying to avoid that, and the timeout works _most_ of the time, though I think if the CI machine is running really slow it still pops up sometimes.  
  
I suspect another reason that our form screenshots seem to be so vulnerable to spurious diffs is also because RJSF rerenders on every update.